core/storage/suggestion/gae_models.py

```diff
@@ -484,13 +484,56 @@ def get_in_review_translation_suggestions(
             list(SuggestionModel). A list of suggestions that are of the given
             type, which are in review, but not created by the given user.
         """
+
         return cls.get_all().filter(datastore_services.all_of(
             cls.status == STATUS_IN_REVIEW,
             cls.suggestion_type == feconf.SUGGESTION_TYPE_TRANSLATE_CONTENT,
             cls.author_id != user_id,
             cls.language_code.IN(language_codes)
         )).fetch(feconf.DEFAULT_SUGGESTION_QUERY_LIMIT)
 
+    @classmethod
+    def get_in_review_translation_suggestions_with_topic_name_and_exp_ids(
+            cls, user_id: str, language_codes: List[str],
+            exp_ids: List[str], topic_name: str | None
+    ) -> Sequence[GeneralSuggestionModel]:
+        """Gets all translation suggestions which are in review.
+
+        Args:
+            user_id: str. The id of the user trying to make this query.
+                As a user cannot review their own suggestions, suggestions
+                authored by the user will be excluded.
+            language_codes: list(str). The list of language codes.
+            exp_ids: list(str). Exploration IDs matching the target ID of the
+                translation suggestions. If the list is empty and a valid
+                topic name is specified, we return an empty list.
+                If the list is empty, and topic name is not specified either,
+                we return all translation suggestions.
+            topic_name: str or None. The name of the topic for which suggestions
+                are being fetched. If topic name is not specified, we fetch all
+                translation suggestions.
+
+        Returns:
+            list(SuggestionModel). A list of suggestions that are of the given
+            type, which are in review, not created by the given user and
+            specific to the exploration IDs passed.
```

Comment:
I have deliberately not added more explanation here, since the same is present while explaining the input args (exp_ids) as well. LMK though if you have some other thoughts around this?

Reply:
I think it's fine, maybe just reword as "...and correspond to the given exploration IDs."

I agree that the longer explanation of "why" only needs to be in the args.

Reply:
I added the extra words.